src/HypercertExchangeClient.ts

```diff
@@ -351,7 +319,7 @@ export class HypercertExchangeClient {
    * @param maker Maker order
    * @param taker Taker order
    * @param signature Signature of the maker order
-   * @param merkleTree If the maker has been signed with a merkle tree
+   * @param merkleTree Optional merkle tree
```

Comment:
Why would the user use a Merkle tree? I think that would be a much better parameter description.

Reply:
I had a more in-depth look, and I originally misinterpreted what the `merkleTree` was referring to. It seems to only refer to the proofs that are obtained when using `signMultipleMakerOrders()` to create the orders. See https://docs.looksrare.org/developers/sdk/signMultipleMakerOrders for a more thorough explanation.

I thought we would use this for allowlist orders, but they are implemented using a strategy and the merkle tree proofs for the allowlist are provided using the `additionalParams` argument. So it looks like we will only need this in the foreseeable future, if we plan on supporting signing multiple sales (and buys) at once from the frontend. That would also include implementing the endpoint for storing the tree and retrieving proofs (see https://docs.looksrare.org/developers/sdk/signMultipleMakerOrders again).

We could decide to remove the functionality, and that means we can get rid of all `merkleTree` parameters in `HypercertExchangeClient`. It would be a breaking change however, as the function signatures would change, but I think it might be worth it.

Looping in @bitbeckers and @holkexyz to weigh in on whether we are going to need this functionality in the future.